aten/src/ATen/Parallel.h

```diff
@@ -30,68 +25,28 @@ AT_API void init_tbb_num_threads();
 constexpr int64_t TBB_GRAIN_SIZE = 32768;
 } // namespace internal
 
-template <class T, template <class> class OP>
-T parallel_reduce(
```

Comment:
Are those really unused now? What did we replace them with?

Reply:
I'll reimplement them later on when working through ReduceOpsKernel.

Reply:
Yes they are unused. At some point ReduceOpsKernel was rewritten with it's own local version of this. I'll undo all of that once I sift through the reductions again in context of also implementing max and min.